code/modules/spells/spell_types/aimed.dm

```diff
@@ -129,6 +129,24 @@
 	projectile_var_overrides = list("range" = range)
 	return ..()
 
+/obj/effect/proc_holder/spell/aimed/grease
+	name = "Grease"
+	desc = "This spell launches a blob of grease at a target, slipping everyone in a 5 tile area around the landing point."
```

Comment:
you should mention in this description that the slippery tiles from grease stick around for 10 seconds (and whether or not wizard sandals will make you immune to slipping on them).